exporter/sumologicexporter/config.go

```diff
@@ -73,6 +73,10 @@ func createDefaultClientConfig() confighttp.ClientConfig {
 
 func (cfg *Config) Validate() error {
 
+	if cfg.ClientConfig.Timeout < 0 || cfg.ClientConfig.Timeout > maxTimeout {
+		return fmt.Errorf("timeout must be between 0 and 55 seconds, got %v", cfg.ClientConfig.Timeout)
+	}
```

Comment:
The underlying dependency that's being used is go's `http.Client` which is [created from this client config](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/73e67ba425481b94f51793df503b2e2b2e619514/exporter/sumologicexporter/exporter.go#L230), from what I can tell.

According to [documentation](https://pkg.go.dev/net/http#Client): 
```
	// A Timeout of zero means no timeout.
```
Should we explicitly not allow 0 to be a valid timeout then, if we want 55 seconds to be the max?

Reply:
ohh good catch, yeah that would be better. will make it between 1 to 55 seconds instead